data/human/human missions.txt

```diff
@@ -7453,3 +7453,233 @@ mission "Adelita 5"
 			`Ale, River, and Bunker look glad to be back on <planet>. "Being there in person was bad enough," Ale grumbles as she looks through a stack of folders on her datapad. "Now we have to watch it over and over again to edit it into something presentable."`
 			`	As they are leaving, Ale hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you. We really couldn't have done it without you," she tells you with a bittersweet smile. "Of course, you'll be one of the first to know when it's done."`
 			`	She starts off to join River and Bunker out in the spaceport before turning around one last time in the doorway. "We might be in need of you for some future projects too, Captain <last>. But goodbye for now. Stay safe on the star lanes!"`
+
+mission "Syndicate Business 5.1 A 10m Expansion Update"
+	landing
+	to offer
+		has "Syndicate Business 5: done"
+		has "event: 10m investment done"
+		or
+			has "Syndicate Business expansion 10m good"
+			has "Syndicate Business expansion 10m neutral"
+			has "Syndicate Business expansion 10m evil"
+	source
+		near Ruchbah 0 20
+	on offer
+		conversation
+			`As you are landing your <model>, you see a message from Howard, the Megaparsec salesman who sold you two investments. The message says:`
+			branch meanopening
+				"syndicate business: mean to Howard" > 0
+			`	"Hey <first>,`
+			`		"I hope all is well. I'm excited to send you this interim Megaparsec report! I'm sharing because it's the first one with your investment included--check out Section 3, Part C, Paragraph 6 for the details."`
+				goto openingchoice
+			label meanopening
+			`	"Captain <first> <last>,`
+			`		"Here's the interim Megaparsec report, sent because your investment is included for the first time. You can find the relevant information in Section 3, Part C, Paragraph 6."`
```

Comment:
```suggestion
			`		"Here's the interim Megaparsec report, sent because your investment is included for the first time. You can find the relevant information in Section 3, Part C, Paragraph 6.`
```